guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,230 @@
+[id="upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL} 9 in-place by using Leapp
+
+{Project} and {SmartProxy} are supported on both {EL}8 and {EL}9.
+You can upgrade your {Project} and {SmartProxy} operating system from {EL} 8 to {EL} 9. 
+When upgrading your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9, use the Leapp tool to upgrade as well as to help detect and resolve issues that could prevent you from upgrading successfully.
+
+.Prerequisites
+ifdef::satellite[]
+* Review Known Issues before you begin an upgrade.
+For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+endif::[]
+* {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
+ifndef::satellite[]
+* Access to available repositories or a local mirror of repositories.
+endif::[]
+
+ifeval::["{mode}" == "disconnected"]
+* You require access to {RHEL} and {Project} packages.
+Obtain the ISO files for {RHEL}{nbsp}9 and {Project}.
+For more information, see xref:Upgrading_satellite_upgrading-disconnected[].
+* Since Leapp completes part of the upgrade in a container that has no access to additional ISO mounts, the repositories cannot be served from a locally mounted ISO but must be delivered over the network from a different machine.
```

Comment:
Do we have a procedure how to set up the repository server on a machine? We should link it here.

Reply:
I think we can refer to this: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/performing_a_standard_rhel_9_installation/assembly_preparing-for-your-installation_installing-rhel#creating-a-remote-repository_creating-a-remote-repository